source/includes/steps-install-shell-base.yaml

```diff
@@ -18,42 +18,6 @@ content: |
    system's OpenSSL installation. For details, see the `MongoDB Download
    Center <https://www.mongodb.com/try/download/shell?jmp=docs>`__.
 ---
-title: "Extract the files from the downloaded archive."
```

Comment:
This was only used in one place so I moved it there to avoid a layer of indirection